pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -203,7 +203,6 @@ class LightningTensor(Device):
         cutoff (float): The threshold used to truncate the singular values of the MPS tensors. Default is 0
         cutoff_mode (str): Singular value truncation mode. Options: ["rel", "abs"]. Default is "abs".
```

Comment:
```suggestion
        cutoff_mode (str): Singular value truncation mode. The options are ``"rel"`` and ``"abs"``. Default is ``"abs"``.
```

Reply:
I don't think it is necessary now, but this explanation can potentially be expanded in the final version.